R/datamation_sanddance.R

```diff
@@ -264,20 +264,34 @@ datamationSandDance_html <- function(...) {
         class = "control-bar",
         shiny::tags$div(
           class = "button-wrapper",
+          shiny::tags$button(onclick = htmlwidgets::JS(paste0("window.", app_name, ".play('1')")), "Gif")
```

Comment:
This errors. Specifically needs commas between shiny::tags(). I updated on my side for testing.

Reply:
Also realizing the error I mention below is because this is calling "play" instead of export. I will test with something like this:

          shiny::tags$button(onclick = htmlwidgets::JS(paste0("window.", app_name, ".exportGif(true)")), "Gif"),